### PR TITLE
community: Resign team lead

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

**Comment by Mixih** [comment]
@sims1253 could you provide a statement why you are stepping down?

**Comment by sims1253** [comment]
I can't put the time into this that I feel is needed to make it worthwhile.
I also feel like I'm not really inspiring anyone to work on anything.
I want to thank all of you for the support you gave me regardless.

**Comment by sims1253** [comment]
Also:
The initial idea was to run the first period for only half a year, as a test. So I almost kinda am there.

**Comment by sims1253** [comment]
And finally, there is a small part about it in the community channel: :point_up: [9. Mai 2017 10:22](https://gitter.im/coala/community?at=59117c299d90dc7a1c5e14bb)

File: community/team_members.md [hunk 1]
```diff
@@ -70,4 +57,22 @@ next month.
...
+I joined coala in early 2015 but real contributions started 2016. I mainly
+work on coala by giving input on design questions helping with administrative
+tasks. My biggest impact on coala so far probably was the community work during
+the second GSoC coala participated in and my conference visits. From that time
+the title Secretary of Offtopic stems."
+
+Max was responsible for:
+
+- Organizing the team
+- GSoC administering
```
Comment (jayvdb) [inline community/team_members.md:78]: so are you resigning from this responsibility?
And it looks like you are resigning from the team, not just the team lead like the commit shortlog/PR title says

Comment (sims1253) [reply community/team_members.md:78]: I am resigning from the team lead of the community team and effectively leaving the team for the time being.
I will keep the GSoC admin responsibility (would have talked to you about that wayyy in advance).

**Comment by shreyans800755** [comment]
ack b52c175

**Pull request merged**